archivebox/main.py

```diff
@@ -252,7 +254,8 @@ def init(force: bool=False, out_dir: str=OUTPUT_DIR) -> None:
     """Initialize a new ArchiveBox collection in the current directory"""
     os.makedirs(out_dir, exist_ok=True)
     is_empty = not len(set(os.listdir(out_dir)) - ALLOWED_IN_OUTPUT_DIR)
-    existing_index = os.path.exists(os.path.join(out_dir, JSON_INDEX_FILENAME))
+
+    existing_index = (Path(out_dir) / SQL_INDEX_FILENAME).exists()
```

Comment:
Can you add a check here for `JSON_INDEX_FILENAME`, it it exists show a warning saying it will no longer be kept up-to-date automatically, and that they should run `archivebox list --json > ...` manually when they want to use this file.

Reply:
https://github.com/pirate/ArchiveBox/pull/452/commits/ee5ba40c831800b83ce2d1e8a82f2cd65422f890